### PR TITLE
renaming

### PR DESCRIPTION
**Pull request by afomi**
This PR updates basic references to `panda` with `yours`.

Screenshot 2024-03-07 at 3 15 25 AM

- this feature branch can be `squash merged`
- each commit is discrete, for feedback and the ability to cherry-pick

#### Notes

* There are still 43 references to `panda`.
* 2 references to https://panda-wallet.gitbook.io/
* 2 references to https://github.com/orgs/Panda-Wallet/projects/1
* A reference to https://panda-wallet-sample-app.vercel.app/
* A reference to https://opencollective.com/panda-wallet
* A reference to https://docs.scrypt.io/tokens/tutorials/ordinal-lock/#use-panda-wallet.
* A reference to https://chrome.google.com/webstore/detail/panda-wallet/mlbni...

### CODE REVIEW

File: src/components/YoursHead.tsx [hunk 1]
```diff
@@ -55,11 +55,11 @@ export const PandaHead = (props: PandaHeadProps) => {
...
 
   return (
-    <PandaImg
+    <Yoursmg
       ref={imgRef}
-      src={panda}
+      src={yoursLogo}
       width={width}
-      alt="Panda Head"
+      alt="Yours Head"
```
Comment (danwag06) [inline src/components/YoursHead.tsx:62]: Yours Icon

File: src/components/YoursHead.tsx [hunk 2]
```diff
@@ -55,11 +55,11 @@ export const PandaHead = (props: PandaHeadProps) => {
   }, [animated]);
 
   return (
-    <PandaImg
+    <Yoursmg
       ref={imgRef}
-      src={panda}
+      src={yoursLogo}
```
Comment (danwag06) [inline src/components/YoursHead.tsx:60]: The source of this image should be updated to use the Yours Logo attached.
yours-logo

File: src/pages/onboarding/ImportWallet.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { BackButton } from '../../components/BackButton';
 import { Button } from '../../components/Button';
 import { Input } from '../../components/Input';
 import { PageLoader } from '../../components/PageLoader';
-import { PandaHead } from '../../components/PandaHead';
+import { YoursHead } from '../../components/YoursHead';
```
Comment (danwag06) [inline src/pages/onboarding/ImportWallet.tsx:8]: `YoursIcon`

File: src/pages/onboarding/ImportWallet.tsx [hunk 4]
```diff
@@ -236,7 +236,7 @@ export const ImportWallet = () => {
   const successStep = (
     <>
       <Content>
-        <PandaHead />
+        <YoursHead />
```
Comment (danwag06) [inline src/pages/onboarding/ImportWallet.tsx:239]: `<YoursIcon />`

File: src/utils/constants.ts [hunk 5]
```diff
@@ -19,10 +19,10 @@ export const P2PKH_OUTPUT_SIZE = 34;
 export const DUST = 10;
 export const INACTIVITY_LIMIT = 10 * 60 * 1000; // 10 minutes
 export const SNACKBAR_TIMEOUT = 3 * 1000; // 2.5 seconds
-export const HOSTED_PANDA_IMAGE = 'https://i.ibb.co/3fLL5X2/Panda-Wallet-Logo.png';
-export const PANDA_DEV_WALLET = '1MtzWXQEYGp89bQ9U2nfrnuChFv37j6pV6';
+export const HOSTED_YOURS_IMAGE = 'https://i.ibb.co/3fLL5X2/Panda-Wallet-Logo.png';
```
Comment (danwag06) [inline src/utils/constants.ts:22]: Update to use the yours icon attached in one of the other comments

**Review by danwag06 — CHANGES_REQUESTED** [review]
Overall this is hugely helpful.

However, we need to make sure these changes are synced with:
https://github.com/Panda-Wallet/panda-wallet-provider

IMPORTANT: things must remain backwards compatible. If you are going to tackle the provider part too, make sure all the types are in sync. Fine with bumping the version of the provider version with a MAJOR change since it would break things. People would just need to update their types if upgrading to v3.0.0 of the provider.

In general, i'm struggling with best way to move all existing issues from the origin Panda repo and have everyone just start working from the Yours fork. Would be nice to migrate all existing issues and this project Kanban if possible https://github.com/orgs/Panda-Wallet/projects/1

Ideally the panda repo just becomes deprecated in favor of the yours repo since we are expecting all the action to move here.

File: src/pages/onboarding/ImportWallet.tsx (hunk 3 above)
Comment (afomi) [reply src/pages/onboarding/ImportWallet.tsx]: https://github.com/afomi/yours-wallet/commit/787a50cf04e9db58f776c2ff148acc51b46e632e

File: src/pages/onboarding/ImportWallet.tsx (hunk 4 above)
Comment (afomi) [reply src/pages/onboarding/ImportWallet.tsx]: https://github.com/afomi/yours-wallet/commit/787a50cf04e9db58f776c2ff148acc51b46e632e

File: src/utils/constants.ts (hunk 5 above)
Comment (afomi) [reply src/utils/constants.ts]: https://github.com/afomi/yours-wallet/commit/811d296114177e6caa404483e31de5f9a3f0ed12

**Comment by afomi** [comment]
updates made to address the comments,
and provider branch is a WIP at https://github.com/Panda-Wallet/panda-wallet-provider/pull/7

**Pull request merged**